packages/ember/tailwind-webpack/index.js

```diff
@@ -0,0 +1,52 @@
+// @ts-check
+import { files, html, packageJson, project } from 'ember-apply';
+import path, { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+// @ts-ignore
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export default async function run() {
+  await packageJson.addDevDependencies({
+    'tailwindcss': '^3.0.0',
+    'postcss': '^8.0.0',
+    'postcss-loader': '^7.0.0',
+    'autoprefixer': '^10.0.0'
+  });
+
+  // TODO: For some reason the css file in the [official guides](https://tailwindcss.com/docs/guides/emberjs) goes to `app/app.css` instead of `app/styles/app.css`. Do we want to honour that?
+  // TODO: I think we should move the `postcss.config.js and tailwind.config.js to app/config, but how to make postcss/tailwind then find them?
```

Comment:
the postcss config file can be specified below -- I'm not sure about the tailwind location, unless `tailwindcss` (the postcss plugin) can have a config option to pass its config -- which i I think it does? you may need to look at the docs for the postcss tailwind plugin for that because I'm not sure